Makefile

```diff
@@ -16,7 +16,8 @@ all: kops
 
 .PHONY: channels examples
 
-DOCKER_REGISTRY?=gcr.io/must-override
+#DOCKER_REGISTRY?=gcr.io/must-override
+DOCKER_REGISTRY=prashima
```

Comment:
Removed my changes. Developers need to modify this part locally in their Makefile copy.